saleor/site/models.py

```diff
@@ -65,6 +65,7 @@ class SiteSettings(models.Model):
     default_mail_sender_address = models.EmailField(blank=True, null=True)
     customer_set_password_url = models.CharField(max_length=255, blank=True, null=True)
     automatically_confirm_all_new_orders = models.BooleanField(default=True)
+    reserve_stock_duration_minutes = models.IntegerField(blank=True, null=True)
```

Comment:
Is it ok that we by default assume that staff user will use minutes here? What if I would like to have 45 seconds for reservation (yes, i know that this is very short time to finish a checkout). What if user will want to have 5.5 minutes?

Reply:
I've stopped doing granularity like that when I've realized scenarios like this only happen during design discussions. In 5.5m scenario user is more likely to write either 5 or 6 instead of going "I'll use different e-commerce solution that lets me enter 330s" 😄

Reply:
👍